public/data/benjaminmichaelis.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "Benjamin Michaelis",
+  "type": "personal",
+  "bio": "Full time student, part-time adventure seeker/traveler",
+  "avatar": "https://github.com/benjaminmichaelis.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/benjaminmichaelis",
+      "icon": "github"
+    },
+    {
+      "name": "Connect with me everywhere!",
+      "url": "https://benjamin.michaelis.net",
+      "icon": "Globe"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/benjamin-michaelis",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/benmichaelis",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Coding",
+      "date": "January 2020",
+      "icon": "code",
+      "color": "#gray",
```

Please use the correct **_Hex code_** for color!

```suggestion
      "color": "#787777",
```